doc/engines.rst

```diff
@@ -67,6 +67,13 @@ Stream
     This should not be confused with a CUDA stream, which corresponds to a
     Command Queue in OpenCL terminology.
 
+Stream Group
+    A group of incoming stress whose data are combined in chunks (see
+    :class:`spead2.recv.ChunkStreamRingGroup`). Stream groups can be logically
+    treated like a single stream, and are used to scale receiving across
```

Comment:
"are" used or "can be" used?
IIRC you're only giving the receive a single thread for the current case? Or did I misunderstand something?

Reply:
For high-bandwidth testing I'm using 4 streams. But let me try to reword it to better capture what I'm trying to say.